beacon-chain/blockchain/process_block.go

```diff
@@ -501,6 +501,15 @@ func (s *Service) handleEpochBoundary(ctx context.Context, postState state.Beaco
 		if err := helpers.UpdateProposerIndicesInCache(ctx, copied); err != nil {
 			return err
 		}
+
+		if err := copied.SetSlot(copied.Slot() + params.BeaconConfig().SlotsPerEpoch); err != nil {
```

Comment:
If slot 31 is missed then, we would skip this extra step. Do we want to handle that case too ?